open-sauced/intro
# fix: Update UI component for improved design

## Description



This PR updates the UI of the card component to improve layout and responsiveness. Additionally, the button within the card is positioned at the bottom for better usability and visual appeal. Please note that the deploy preview link is currently not working, but the changes have been tested locally and are functioning as expected.

## Related Tickets & Documents


Closes #243

## Mobile & Desktop Screenshots/Recordings



**Before**
![Before](https://github.com/user-attachments/assets/ea95ce14-cab0-4f72-a8b6-641783fc0522)

**After**
![After](https://github.com/user-attachments/assets/8740e3ae-b5d9-4a8d-a109-9a22a9d58d6c)


## Steps to QA
 

1. Pull the latest changes.
2. Run the project locally.
3. Verify that the card layout is responsive and the button is correctly positioned at the bottom.
4. Ensure no issues arise with other components.

## Tier (staff will fill in)


## [optional] What gif best describes this PR or how it makes you feel?
